src/common/store/pdfFiles/actions.js

```diff
@@ -9,7 +9,11 @@ export const addPdfFile = createAction('PDF_FILE_ADD', ({ file, hasBackup = fals
     hasBackup,
 }));
 
-export const updatePdfFile = createAction('PDF_FILE_UPDATE', ({ id, fileName }) => ({ id, name: fileName }));
+export const updatePdfFile = createAction('PDF_FILE_UPDATE', ({ id, fileName, contentSize }) => ({
+    name: fileName,
+    size: contentSize,
+    id,
+}));
 
 export const storeFile = file => async dispatch => {
     const hasBackup = await setFile(file);
```

Comment:
Please reuse the method below `storeFileWithLink` (aka `saveFileToStorage`) here.